grails-shell/build.gradle

```diff
@@ -62,18 +62,17 @@ dependencies {
         exclude group: "org.slf4j", module:"jcl-over-slf4j"
         exclude group: "commons-codec", module:"commons-codec"
```

Comment:
No need to exclude `commons-codec:commons-codec`

Reply:
removed